src/modules/products/products.service.ts

```diff
@@ -202,4 +206,41 @@ export class ProductsService {
       data: {},
     };
   }
+
+  async addCommentToProduct(productId: string, commentDto: AddCommentDto, userId: string) {
+    const { comment } = commentDto;
+    const product = await this.productRepository.findOne({ where: { id: productId } });
+    const user = await this.userRepository.findOne({ where: { id: userId } });
+
+    if (!product) {
+      throw new NotFoundException({
```

Comment:
Use customHttpExceptions provided in the helper directory. reference auth service for guide on usage

Reply:
Done